Wiki/FAQ.md

```diff
@@ -0,0 +1,709 @@
+chain you will be paid for the research you are owed. This is what makes
+this method so much more fair, than the old PoW system. With
+Proof-of-Research as long as you find enough blocks in a 2 week window
+to be paid your full reward within the payout rules you will receive all
+the rewards you earned in the same period.
+
+## What should I put in my config-file for Gridcoin-Research?
+
+Please find gridcoinresearch.conf in your
+%appdata%\\roaming\\gridcoinresearch folder and insert a line for your
+BOINC email address if it is empty:
+
+`email=YourBoincEmail`  
+`addnode=node.gridcoin.us`
+
+These lines should already suffice. Else, find more at [GridcoinResearch
+Config
+file](Gridcoinresearch-Config-File "wikilink")
+
+## How can I tell if I've mined a block?
+
+First look at the transaction list for "PoR" payments.
+
+If you don't have any PoR yet, do a "list mymagnitude" in the debug
+console and look for the savings account balance "owed". That will be
+what you accrue for your first payment. It has to be higher than the
+subsidyschedule max block / 10, IE 40 right now in order to actually
+generate.
+
+## Reward calculation: what is RAC?
+
+<https://en.wikipedia.org/wiki/BOINC_Credit_System#Recent_average_credit>
+
+## Reward calculation: what is magnitude?
+
+The reward calculation is based on the relative contribution of a user
+to the total contribution of team Gridcoin. This is being calculated
+within every project and the sum of the relative contributions is the
+magnitude which is being used to calculate the reward. Besides the
+research rewards, each mined block contains interest.
+
+## Is it possible to join a pool?
+
+Yes, there are pools at <https://grcpool.com/> and
+<https://grc.arikado.ru/>
+
+## I am BOINCing nonstop but my magnitude is very low. What can I do?
+
+Regarding your magnitude: You should have an eye on your RAC and the
+project total RAC within team Gridcoin. Be aware that BOINC RAC is
+measured weekly, with the most recent week being higher weighted, the
+next week being weighted at half of the recent week and so on, it
+therefore takes a number of weeks of consistent BOINC work on a
+specified project, for RAC to increase to sustainable levels.
+
+Have a look here:
+<http://www.gridresearchcorp.com/gridcoin/?result&t=Whitelisted_Project_Leaderboard>
+. If your hardware is not strong or compatible, you should keep away
+from some projects. For example, GPUGRID is very popular for high end
+GPUs so the team RAC is high. My GTX 770 got me RAC way above 10k but
+the magnitude was smaller than 10. So I decided to let others work on
+that project.
+
+There are also overrun CPU projects or projects that do not provide
+constant work unit (WU) flow.
+
+Try different projects and see whether your hardware can keep up a high
+magnitude. You can review your outcome magnitude in the Neural Network.
+Go to Advanced -\> Neural Network and search for your CPID. Then you can

```

Comment:
Same deal about the Neural Network stuff. Might be able to axe this whole paragraph